lib/autobox/Transform.pm

```diff
@@ -516,6 +717,71 @@ sub filter {
     return wantarray ? @$result : $result;
 }
 
+# order has comparisons has options
+sub order {
+    my $array = shift;
+    my ($options) = @_;
+    ref($options) eq "ARRAY" or $options = [ $options ];
+
+    my $option__group = {
+        str  => "operator",
+        num  => "operator",
+        asc  => "direction",
+        desc => "direction",
+    };
+
+    # Check one comparison
+    my $group__value = {};
+    for my $option (grep { $_ } @$options) {
+        my $group;
+        ref($option) eq "CODE"   and $group = "extract";
+        if( ref($option) eq "Regexp" ) {
+            my $regex = $option;
+            $option = sub { join("", m/$regex/) };
+            $group = "extract";
+        }
+
+        $group ||= $option__group->{ $option }
+            or Carp::croak("->order(): Invalid comparison option ($option)");
+
+        exists $group__value->{ $group }
+            and Carp::croak("->order(): Conflicting comparison options: ($group__value->{ $group }) and ($option)");
+
+        $group__value->{ $group } = $option;
+    }
+
+    my $operator  = $group__value->{operator}  // "str";
+    my $direction = $group__value->{direction} // "asc";
+    my $extract   = $group__value->{extract}   // sub { $_ };
+
+    my $t__ms = {
```

Comment:
transform --> Sort::Maker (see, it's not even named correctly, let alone that it's abbreviated.

Will spell it out.